docs/documentation/case.md

```diff
@@ -331,6 +333,10 @@ Note that `time_stepper` $=$ 3 specifies the total variation diminishing (TVD),
 - `wave_speeds` specifies the choice of the method to compute the left, right, and middle wave speeds in the Riemann solver by an integer of 1 and 2.
 `wave_speeds` $=$ 1 and 2 correspond to the direct method ([Batten et al., 1997](references.md#Batten97)), and indirect method that approximates the pressures and velocity ([Toro, 2013](references.md#Toro13)), respectively.
 
+- `weno_Re_flux` activates the scaler divergence theorem in computing the velocity gradients using WENO-reconstructed cell boundary values.
```

Comment:
add text here that says that if this is _false_ then the code will use finite differences of order X (please fill in what X is).

Reply:
X depends on? .... order of weno? fd_coef? ?